src/javascript/analytics.js

```diff
@@ -66,16 +67,14 @@ export const NULL_VALUE = '(not set)';
  *     `trackingId` fields.
  * @return {Function} The proxied ga() function.
  */
-const createGaProxy = (trackers) => {
-  return (command, ...args) => {
-    for (let {name} of trackers) {
-      if (typeof command == 'function') {
-        window.ga(() => command(window.ga.getByName(name)));
-      } else {
-        window.ga(`${name}.${command}`, ...args);
-      }
+const createGaProxy = trackers => (command, ...args) => {
+  for (const {name} of trackers) {
+    if (isFunction(command)) {
```

Comment:
Ah. I'm used to using extension for webpack (or maybe babel, I don't recall) that counteracts that, and automatically trims lodash imports to include only the stuff you explicitly import. It actually produces *smaller* binaries than doing the specific path'd import. Fixed.

I prefer isFunction because it's doing more stuff than just `typeof === function` ((source)[https://github.com/lodash/lodash/blob/4.17.4/lodash.js#L11661]), which implies to me that there's some kind of javascript nonsense you have to account for in order for it to be "correct". As a rule, I generally prefer the lodash type checking utilities for that reason.

Reply:
In general I don't have a problem with using this lodash methods if we're already using other lodash methods.

For some context, that file is based on my [analyticsjs-boilerplate](https://github.com/philipwalton/analyticsjs-boilerplate) project, and I like to keep it as close to the source as possible, so when I make upgrades to the boilerplate it's easy to upgrade them here as well.

As for your other comments. I get that concern, but I think most of the fear of JavaScript I hear these days is overblown. For example, Lodash's `isFunction()` handles cases where it tests against a constructor, but that's literally never going to happen in the above code.

My stance is that as long as your code is well tested and your tests pass in the environments you support, there's no need to use additional caution just out of fear. (Demos & Tools is not well-tested, so that's a different story.)